README.md

```diff
@@ -1,3 +1,8 @@
+# Deprecated
+With the release of SpringRoll 2.0 this repository has been deprecated.
+
+To manage captions you can checkout the new online solution at [springroll.io](http://springroll.io)
```

Comment:
Let's also add a reference to SpringRoll-Seed here too (for stubbing projects).  Basically want to communicate that all the features that SpringRollStudio has are now accomplished by other projects.